v1/linkml-schemas/lifeEvents.yaml

```diff
@@ -0,0 +1,70 @@
+id: https://vocab.account.gov.uk/linkml/life-events-schema
+name: lifeEvents
+version: 1.0
+description: >-
+  Vocabulary for life events
+
+prefixes:
+  linkml: https://w3id.org/linkml/
+  vc: https://www.w3.org/2018/credentials/v1/
+  di_vocab: https://vocab.account.gov.uk/v1/
+imports:
+  - linkml:types
+  - ./events
+  - ./person
+default_curi_maps:
+  - semweb_context
+default_prefix: di_vocab
+default_range: string
+
+classes:
+  DeathRegistrationSETClass:
+    is_a: SETClass
+    attributes:
+      events:
+        range: DeathRegistrationEventMappingClass
+        required: true
+  DeathRegistrationEventMappingClass:
+    attributes:
+      "deathRegistrationUpdate":
+        range: DeathRegistrationEventClass
+        required: true
+        alias: "https://ssf.account.gov.uk/v1/deathRegistration"
+  DeathRegistrationUpdateSETClass:
+    is_a: SETClass
+    attributes:
+      events:
+        range: DeathRegistrationUpdateEventMappingClass
+        required: true
+  DeathRegistrationUpdateEventMappingClass:
+    attributes:
+      "deathRegistrationUpdate":
+        range: DeathRegistrationEventClass
+        required: true
+        alias: "https://ssf.account.gov.uk/v1/deathRegistrationUpdate"
+  DeathRegistrationEventClass:
+    slots:
+      - deathRegistrationID
+      - subject
+      - deathDate
+      - freeFormatDeathDate
```

Comment:
So we reckon that "death registration date" is covered by the SET property `toe` (for death registrations, not for updates, where "death registration date" isn't included anyway.

I think we will need to put a custom description on the `toe` slot/attribute for each event we define - I believe this is possible but haven't attempted it yet.

Reply:
Although thinking about it, this is weird because we don't actually know the time, so that would have to be zero'd out. There's definitely a case for breaking it out into a separate field like you're suggesting.

Reply:
I think there is definitely room for ambiguity if we use `toe` so would prefer a specific field in this case.  It may be possible that the event gets registered at a system datetime that is after the date of registration due to manual data input processes, batch input, legacy reporting mechanisms etc.  So I think that `toe` *could* be seen as the system time the event was registered within the GRO database, but the `deathRegistrationDate` is not guaranteed to be the same and as you say wouldn't have a precise time associated with it.

Reply:
OK cool good argument, I'll commit these suggestions (and squash them, so expect to need to pull the repo down again :) )